### ISSUE TITLE
Set this as update 3.0 on spigot.

### ISSUE THREAD

**Issue by Legoman99573** [SpigotMCUpdater/SpigotMC-Updater#2]
I saw that this is more clear and more understanding. Do you mind if I set updates based on this script as it would support all versions?

**Comment by Legoman99573** [comment]
Would probably add the option to run the buildtools jar and allow users to call a Directory. I will commit the respiratory if you like.

**Comment by NatoBoram** [comment]
Sorry, I don't quite understand what you said, do you mean adding some options to the [BuildTools.sh](https://github.com/SpigotMCUpdater/SpigotMC-Updater/blob/master/BuildTools.sh)? Or modifying [menu.sh](https://github.com/SpigotMCUpdater/SpigotMC-Updater/blob/master/menu.sh) by using  BuildTools.sh's code? BuildTools.sh is already compatible with both Windows, Linux and MacOS. In fact, I am currently running BuildTools.sh with my Debian installation right now!

**Comment by Legoman99573** [comment]
No, make it the next update in the resource i still have up (will leave credit if i remember your username on spigot)

**Comment by NatoBoram** [comment]
You can do as you want ^^
Sorry for the absence, I didn't see/know there was a reply there!
You can actually create a [release](https://github.com/SpigotMCUpdater/SpigotMC-Updater/releases) here, too!
